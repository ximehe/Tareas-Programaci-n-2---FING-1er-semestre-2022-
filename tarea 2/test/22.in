# implementar insertarEnColCadenas
cantidadColCadenas 0
cantidadColCadenas 9
insertarEnColCadenas 3 3.0 0
estaEnColCadenas 3 0
estaEnColCadenas 3 1
insertarEnColCadenas 3 13.1 9
insertarEnColCadenas 3 13.1 5
cantidadColCadenas 0
cantidadColCadenas 1
cantidadColCadenas 5
cantidadColCadenas 9
estaEnColCadenas 3 9
estaEnColCadenas 3 5
insertarEnColCadenas 8 8.0 0
insertarEnColCadenas 9 9.0 0
insertarEnColCadenas 5 5.0 5
insertarEnColCadenas 6 6.0 5
insertarEnColCadenas 7 7.0 5
insertarEnColCadenas 4 4.0 9
cantidadColCadenas 0
cantidadColCadenas 1
cantidadColCadenas 5
cantidadColCadenas 9
estaEnColCadenas 9 0
estaEnColCadenas 9 5
estaEnColCadenas 6 5
estaEnColCadenas 4 0
estaEnColCadenas 4 9
Fin