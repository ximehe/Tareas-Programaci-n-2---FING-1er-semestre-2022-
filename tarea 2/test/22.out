1>#  implementar insertarEnColCadenas.
2>La cantidad en la cadena 0 es 0.
3>La cantidad en la cadena 9 es 0.
4>Se insertó (3,3.00) en la cadena 0.
5>3 está en la cadena 0.
6>3 no está en la cadena 1.
7>Se insertó (3,13.10) en la cadena 9.
8>Se insertó (3,13.10) en la cadena 5.
9>La cantidad en la cadena 0 es 1.
10>La cantidad en la cadena 1 es 0.
11>La cantidad en la cadena 5 es 1.
12>La cantidad en la cadena 9 es 1.
13>3 está en la cadena 9.
14>3 está en la cadena 5.
15>Se insertó (8,8.00) en la cadena 0.
16>Se insertó (9,9.00) en la cadena 0.
17>Se insertó (5,5.00) en la cadena 5.
18>Se insertó (6,6.00) en la cadena 5.
19>Se insertó (7,7.00) en la cadena 5.
20>Se insertó (4,4.00) en la cadena 9.
21>La cantidad en la cadena 0 es 3.
22>La cantidad en la cadena 1 es 0.
23>La cantidad en la cadena 5 es 4.
24>La cantidad en la cadena 9 es 2.
25>9 está en la cadena 0.
26>9 no está en la cadena 5.
27>6 está en la cadena 5.
28>4 no está en la cadena 0.
29>4 está en la cadena 9.
30>Fin.
